src/main/java/org/broadinstitute/hellbender/tools/walkers/variantutils/ValidateVariants.java

```diff
@@ -55,26 +55,26 @@
  *
  * <h4>To perform VCF format and all strict validations: </h4>
  * <pre>
- *   ./gatk-launch ValidateVariants \
+ *   gatk ValidateVariants \
```

Comment:
Looks like this is one of those tools that drops the reference requirement. So the simplest command that can be run is as follows. I suggest putting these two examples at top then follow with whichever example commands seem appropriate, e.g. one that utilizes dbsnp and another to exclude validation types.

Minimally validate a file for adherence to VCF format
```
gatk ValidateVariants \
    -V cohort.vcf.gz
```

Validate a GVCF for adherence to VCF format, including REF allele match
```
gatk ValidateVariants \
    -V sample.g.vcf.gz \
    -R reference.fasta
    -gvcf
```

Looks like there is a bug though for REF allele matching a VCF. While the GVCF validation catches the REF mismatch, using a VCF (while supplying the reference) does not. Hmm, that's disturbing. I didn't get around to testing the `--dbsnp` nor AC/AN values. Why aren't the tests catching this? How should Marton/self proceed with this observation @droazen @vdauwera?